<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->

  <!-- Start yolov5 and ros wrapper -->
  <node pkg="coordinate_system_conversion" type="conversion.py" name="coordinate" output="screen" >
    <param name="coordinate_information_topic"       value="/yolov5/BoundingBoxes" />
    <param name="camera_info_topic"       value="/camera/color/camera_info" />
    <param name="accel_info_topic"       value="/camera/accel/sample" />
    <param name="pub_topic"         value="/coordinate_system_conversion/CoordinatePoint" />
    <param name="pub_flag_topic"         value="/coordinate_system_conversion/CalibrateFlag" />

  </node>

</launch>